appeals/api/src/server/endpoints/appellant-cases/__tests__/appellant-cases.test.js

```diff
@@ -251,7 +251,7 @@ describe('appellant cases routes', () => {
 				);
 
 				const body = {
-					appealDueDate: '2023-07-14',
+					appealDueDate: '2099-07-14',
```

Comment:
Perhaps we could have an additional test to check that the correct error message is returned if `appealDueDate` in the request body is in the past